tests/self_peertest.rs

```diff
@@ -59,10 +59,6 @@ mod test {
             peertest_config.clone(),
             &peertest,
         );
-        peertest::scenarios::eth_rpc::test_eth_get_block_by_number(
-            peertest_config.clone(),
-            &peertest,
-        );
```

Comment:
Just making sure this gets added back when the last part of the test gets added again.